src/session/session_compact.c

```diff
@@ -245,6 +248,16 @@ __compact_worker(WT_SESSION_IMPL *session)
      */
     WT_ERR(__compact_checkpoint(session));
 
+#ifdef HAVE_DIAGNOSTIC
+    /*
+     * Create the compact_started file so that the test can start its timer.
+     */
+    WT_ERR(
+      __wt_snprintf(compact_file, sizeof(compact_file), "%s/compact_started", S2C(session)->home));
```

Comment:
We cannot do this. WT does not own that namespace. The user may have a file of the same name.